### ISSUE TITLE
malware

### ISSUE THREAD

**Issue by jaky9999** [Lsociety1337/NordVPN-Checker#1]
fucking malware

**Comment by LegendaryEmoji** [comment]
Indeed. Scroll horizontally on [Line #1](https://github.com/Lsociety1337/NordVPN-Checker/blob/main/main.py#L1) to see the malicious code.
